thermostat/core.py

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
@EthanGoldman This file wont automatically merge, I want to confirm that we should always take the tau friendly results when merging these